code/src/components/Header.css

```diff
@@ -0,0 +1,46 @@
+* {
+    margin: 0;
+  }
+
+header {
+    height: 200px;
+    padding: 16px;
+    position: relative;
+  }
+
+  .header-accessory {
```

Comment:
Nicely done with the circles (or sun's which was the first word that popped up to me when viewing them), consider giving them more descriptive names? :)